commands/system/update.sh

```diff
@@ -10,6 +10,11 @@ function system:update() {
 
     _logYellow "Updating dde repository"
     cd ${ROOT_DIR}
+    branch=$(git rev-parse --abbrev-ref HEAD)
+    if [ $branch != 'master' ] && [ $branch != 'main' ]
+    then
+        _logYellow "# Be careful! You have not checked out master. You're currently on: $branch"
```

Comment:
@Ruesa18 none of the other warnings is prefixed with a `#`.

Reply:
@s4mpl3d Missed that. Fixed 👍 
I also adapted the message to not explicitly say `master` since I check for both `master` and `main`. It now says. `Be careful! You have not checked out the stable branch for dde. You're currently on: $branch`